react-swipe.js

```diff
@@ -65,9 +65,16 @@
     shouldComponentUpdate: function (nextProps) {
       return (
         (this.props.slideToIndex !== nextProps.slideToIndex) ||
-        (typeof this.props.shouldUpdate !== 'undefined') && !this.props.shouldUpdate(nextProps)
+        (typeof this.props.shouldUpdate !== 'undefined') && !this.props.shouldUpdate(nextProps)||
+        (nextProps.children!==this.props.children)
```

Comment:
I don't think this is a good enough check - there are occasions when the children object will be effectively the same elements but not a reference to the same object, no?